scripts/west_commands/zcmake.py

```diff
@@ -49,8 +49,7 @@ def run_cmake(args, cwd=None, capture_output=False):
             return None
     else:
         # A real error occurred, raise an exception
-        raise subprocess.CalledProcessError(cmd=p.args,
-                                            returncode=p.returncode)
+        raise subprocess.CalledProcessError(p.returncode, p.args)
```

Comment:
Looks good, thanks. I think since that resolves the issue for west v0.5.8 that there's no need to cut a branch, but I'll still fix this in master. OK for you?

Reply:
yes, please do, thanks!